### PR TITLE
Test out transaction sending

### PR DESCRIPTION
**Pull request by pette1999**
Fixed small bugs

### CODE REVIEW

File: example-apps/Token-EasySending/src/web3/account.ts [hunk 1]
```diff
@@ -21,6 +21,14 @@ async function isContract(provider: Provider, address: string) : Promise<boolean
     return false;
 }
 
+export function createNameStruct(input: string, type: "email" | "twitter"): NameStruct {
+    return {
+        schema: hash((type == "email" && "mailto").toString()),
```
Comment (zihaoccc) [inline example-apps/Token-EasySending/src/web3/account.ts:26]: I do not think use "&&" is a good way to determine the logic.
for this case, we do not cover the type = twitter
what if we use something like

```
type = "email" ? "mailto" : "twitter.com"
```

Same for domain and handle

File: example-apps/Token-EasySending/src/components/SendTokenModal.vue [hunk 2]
```diff
@@ -316,7 +315,12 @@ const onSubmit = async (_e: Event) => {
       sendStatus.value = "processing";
       message.value = "Check your wallet to confirm the operation...";
 
-      const target = ethers.utils.getAddress(transaction.value.to);
+      const target = validateEmail(transaction.value.to) && ethers.utils.getAddress(
```
Comment (zihaoccc) [inline example-apps/Token-EasySending/src/components/SendTokenModal.vue:318]: I think target here we can use
```
getAccountAddress()
```
directly to fetch the account onchain address

File: example-apps/Token-EasySending/src/components/SendTokenModal.vue [hunk 3]
```diff
@@ -316,7 +315,12 @@ const onSubmit = async (_e: Event) => {
...
+        await getAccountAddress(
+          createNameStruct(transaction.value.to, "email")
```
Comment (zihaoccc) [inline example-apps/Token-EasySending/src/components/SendTokenModal.vue:320]: May I ask what is the difference between this function and getName() in account.ts

File: example-apps/Token-EasySending/src/web3/account.ts [hunk 4]
```diff
@@ -21,6 +21,14 @@ async function isContract(provider: Provider, address: string) : Promise<boolean
     return false;
 }
 
+export function createNameStruct(input: string, type: "email" | "twitter"): NameStruct {
```
Comment (zihaoccc) [inline example-apps/Token-EasySending/src/web3/account.ts:24]: If we can use getName() fn which is defined above to fetch the user corresponding NameStruct, maybe we do not need this one to create a new Struct by passing in value

File: example-apps/Token-EasySending/src/components/SendTokenModal.vue (hunk 2 above)
Comment (zihaoccc) [inline example-apps/Token-EasySending/src/components/SendTokenModal.vue:318]: if validateEmail === false, target will be a Boolean which is not acceptable for following function

File: example-apps/Token-EasySending/src/web3/account.ts (hunk 1 above)
Comment (pette1999) [reply example-apps/Token-EasySending/src/web3/account.ts:26]: Good point, updated

File: example-apps/Token-EasySending/src/components/SendTokenModal.vue (hunk 2 above)
Comment (pette1999) [reply example-apps/Token-EasySending/src/components/SendTokenModal.vue:318]: fixed

**Pull request merged**